src/StrList.cc

```diff
@@ -49,6 +49,21 @@ strListIsMember(const String * list, const SBuf &m, char del)
     return 0;
 }
 
+bool
+strListIsMember_if(const String *list, char del, MemberFilter filter)
+{
+    const char *pos = NULL;
+    const char *item;
+    int ilen = 0;
+    assert(list);
```

Comment:
If 'list' is never nil here, should we use "const String&" as the function parameter?

Reply:
I just use this form for StrList API consistency. The other StrList* functions uses "const String *" as parameter.
If it is huge problem I will make this change however.